src/central-server/management-service/core/src/main/java/org/niis/xroad/cs/management/core/configuration/ManagementServiceProperties.java

```diff
@@ -1,21 +1,21 @@
 /**
  * The MIT License
- *
```

Comment:
A "hack" for future: u  can switch loging to a simple comment and then intellij would not format it. ( /** -> /* )